services/brig/src/Brig/Team/API.hs

```diff
@@ -302,15 +417,17 @@ suspendTeam ::
     Member TinyLog r,
     Member (Input (Local ())) r,
     Member (Input UTCTime) r,
-    Member (ConnectionStore InternalPaging) r
+    Member (ConnectionStore InternalPaging) r,
+    Member InvitationCodeStore r
   ) =>
   TeamId ->
   (Handler r) NoContent
 suspendTeam tid = do
-  Log.info $ Log.msg (Log.val "Team suspended") ~~ Log.field "team" (toByteString tid)
+  lift $ liftSem $ Log.info $ Log.msg (Log.val "Team suspended") ~~ Log.field "team" (toByteString tid)
   changeTeamAccountStatuses tid Suspended
-  lift $ wrapClient $ DB.deleteInvitations tid
-  lift $ liftSem $ GalleyAPIAccess.changeTeamStatus tid Team.Suspended Nothing
+  lift . liftSem $ do
+    Store.deleteInvitations tid
+    GalleyAPIAccess.changeTeamStatus tid Team.Suspended Nothing
```

Comment:
how is this different from line 311/427 above?

Reply:
That is odd, seems we're calling both the old function and the new subsystem one. I was under the impression we were _deleting_ old functions as we moved to to Polysemy....

Reply:
I'm adding comments. This one changes the team status in Galley, the other one updates all users from the given team.